Singularity/Singularity/platform/PlatformBlank.cs

```diff
@@ -200,7 +200,7 @@ public void Draw(SpriteBatch spritebatch)
         /// <inheritdoc cref="Singularity.property.IUpdate"/>
         public void Update(GameTime t)
         {
-            throw new NotImplementedException();
```

Comment:
why remove it? I mean it is really not implemented yet, I'd leave it there tbh

Reply:
Well yeah your point is definitely true, but it would complicate things with a good architecture, since update gets called by the gamescreen on all its objects, and if we throw an exception well get runtime errors (obviously) but we rather want to know that we need to add code, but not the program crashing. Thus a todo would definitely be more appropriate.